packages/commonwealth/client/scripts/views/pages/search/index.tsx

```diff
@@ -54,6 +55,24 @@ const SearchPage = () => {
   const [urlQueryParams] = useSearchParams();
   const [bottomRef, bottomInView] = useInView();
 
+  const [toggleMobileView, setToggleMobileView] = useState(
+    location.pathname.includes('search') && window.innerWidth <= 425
+  );
+
+  useEffect(() => {
+    const handleResize = () => {
+      setToggleMobileView(
+        location.pathname.includes('search') && window.innerWidth <= 425
+      );
+    };
+
+    window.addEventListener('resize', handleResize);
+
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, [location.pathname]);
```

Comment:
same here, Do Not Repeat your code => this should be util/hook

Reply:
Fixed in next commit